Models/Effects/damage-smoke/smoke.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0"?>
+
+<PropertyList>
+
+    <params>
+        <engine>
+            <damage-smoke>/fdm/jsbsim/engine/damage-level</damage-smoke>
```

Comment:
This doesn't work over MP. Let's use the new 2017.2 protocol:

Can you change this to `<damage-smoke>sim/multiplay/generic/bool[0]</damage-smoke>`.

Then replace the whole `<and>` block in the `<condition>` below with just `<property alias="/params/engine/damage-smoke"/>`.

In the -set.xml file, add to the list of generic properties (at the end of the block):

```xml
<!-- Effects -->
<bool n="0" alias="/aircraft/effects/damage-smoke-visible"/>
```

Then create a JSBSim `<switch>` block somewhere and test for `/fdm/jsbsim/engine/damage-level GT 100` and `/engines/active-engine/running EQ 1`. Make sure to add `<output>/aircraft/effects/damage-smoke-visible</output>`.

Reply:
Also add in the -set.xml file:

```xml
<aircraft>
    <effects>
        <damage-smoke-visible type="bool">false</damage-smoke-visible>
    </effects>
</aircraft>
```